Makefile

```diff
@@ -888,7 +888,7 @@ ifeq ($(MINIKUBE_BUILD_IN_DOCKER),y)
 	test "`strings $@ | grep '^LIBVIRT_[0-9]' | sort | tail -n 1`" = "LIBVIRT_1.2.9"
 else
 	$(if $(quiet),@echo "  GO       $@")
-	$(Q)GOARCH=$* \
+	$(Q)GOARCH=$* GOOS=linux \
```

Comment:
The kvm2 driver needs libvirt for linking, so it doesn't support cross-compiling anyway - thus the GOOS is not needed

Reply:
gotcha, good catch. i was testing something and this change was leftover.